rbtagger.el

```diff
@@ -100,19 +102,47 @@ candidates list, then it loops through the list and calls
 your tags file was parsed with ripper-tags --emacs and --extra=q
 options."
   (interactive
-   (let ((symbol (rbtagger-symbol-at-point)))
-     (when (or current-prefix-arg (string= symbol ""))
-       (setq symbol (completing-read (concat "Find definitions of"
-                                             (if (string= symbol "")
-                                                 ""
-                                               (concat " (" symbol ")"))
-                                             ": ")
-                                     (xref-backend-identifier-completion-table 'etags)
-                                     nil nil nil
-                                     'xref--read-identifier-history
-                                     symbol))
-       (if (string= symbol "") (error "Please, specify a symbol!")))
-     (list symbol)))
+   (rbtagger--interactive-get-symbol current-prefix-arg))
+  (rbtagger--find-definitions symbol :same-window))
+
+;;;###autoload
+(defun rbtagger-find-definitions-other-window (symbol)
+  "Find definitions for the Ruby SYMBOL at point in another window.
+See `rbtagger-find-definitions'."
+  (interactive
+   (rbtagger--interactive-get-symbol current-prefix-arg))
+  (rbtagger--find-definitions symbol :other-window))
+
+;;;###autoload
+(defun rbtagger-find-definitions-other-frame (symbol)
```

Comment:
Batch mode has some issues with frames, so I could not yet find a way to test this...